src/aiohomeconnect/client.py

```diff
@@ -678,26 +714,21 @@ async def get_all_events(
         """
         response = await self._auth.request(
             "GET",
-            "/homeappliances/events",
-            headers={"Accept-Language": accept_language},
+            f"/homeappliances/{ha_id}/events",
+            headers={
+                "Accept-Language": accept_language,
+                "Accept": "application/vnd.bsh.sdk.v1+json",
+            },
         )
         return ArrayOfEvents.from_dict(response.json()["data"])
 
-    async def get_events(
+    @asynccontextmanager
+    async def stream_all_events(
         self,
-        ha_id: str,
         *,
         accept_language: Language | None = None,
-        accept: ContentType | None = None,
-    ) -> ArrayOfEvents:
-        """Get stream of events for one appliance - NOT WORKING WITH SWAGGER.
-
-        If you want to do a one-time query of the current status, you can ask for
-        the content-type `application/vnd.bsh.sdk.v1+json` and get the status
-        as normal HTTP response.
-
-        If you want an ongoing stream of events in real time, ask for the content
-        type `text/event-stream` and you'll get a stream as Server Sent Events.
+    ) -> AsyncIterator[EventSource]:
```

Comment:
This method should yield a Home Connect API model. Ie we should not yield the raw SSE model.